danger-kotlin-sdk/src/main/kotlin/systems/danger/kotlin/sdk/DangerKotlinAPI.kt

```diff
@@ -10,11 +10,22 @@ interface DangerContext {
     fun fail(message: String)
     fun fail(message: String, file: String, line: Int)
     fun suggest(code: String, file: String, line: Int)
+
+    val fails: List<Violation>
+    val warnings: List<Violation>
+    val messages: List<Violation>
+    val markdowns: List<Violation>
+}
+
+interface Violation {
```

Comment:
Nitpick: what about IViolation for the interface and Violation for the implementation?